observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -12,6 +12,107 @@ The Network Observability Operator enables administrators to observe and analyze
+
+[id="network-observability-operator-tcp-flags-filtering-1-7_{context}"]
+==== TCP flags filtering
+You can now use the `tcpFlags` filter to limit the volume of packets processed by the eBPF program. For more information, see xref:../../observability/network_observability/observing-network-traffic.adoc#network-observability-flowcollector-flowfilter-parameters_nw-observe-network-traffic[Flow filter configuration parameters] and xref:../../observability/network_observability/observing-network-traffic.adoc#network-observability-ebpf-flow-rule-filter_nw-observe-network-traffic[eBPF flow rule filter].
+
+[id="network-observability-cli-1-7_{context}"]
+==== Network Observability CLI
+The Network Observability CLI (`oc netobserv`), is now generally available. The following enhancements have been made since the 1.6 Technology Preview release:
+
+* There are now eBPF enrichment filters for packet capture similar to flow capture.
+* You can now use filter `tcp_flags` with both flow and packets capture.
+* The auto-teardown option when max-bytes or max-time is hit 
+
+For more information, see xref:../../observability/network_observability/netobserv_cli/netobserv-cli-install.adoc#network-observability-netoberv-cli-about_netobserv-cli-install[Network Observability CLI]
+
+[id="network-observability-virtualization_{context}"]
+==== Network Observability for OpenShift Virtualization
+You can observe networking patterns on an OpenShift Virtualization setup by identifying eBPF-enriched network flows coming from VMs that are connected to secondary networks, such as through Open Virtual Network (OVN)-Kubernetes. For more information, see 
+// link to ../../observability/network_observability/network-observability-secondary-networks.adoc#network-observability-virtualization-config_secondary_networks[Configuring virtual machine (VM) secondary network interfaces for Network Observability]
+// 
+
+[id="network-observability-network-policy-1-7_{context}"]
+==== Network policy deploys in the FlowCollector custom resource (CR)
+With this release, you can configure the `FlowCollector` CR to deploy a Network policy for Network Observability. Previously, if you wanted a network policy, you had to manually create one. The option to manually create a network policy is still available. For more information, see 
+
+// link to ../../observability/network_observability/network-observability-network-policy[Configuring an ingress network policy by using the FlowCollector custom resource]
+
+[id="network-observability-fips-compliance-1-7_{context}"]
+==== FIPS compliance
+* You can install and use the Network Observability Operator in an {product-title} cluster running in FIPS mode.
++
+--
+include::snippets/fips-snippet.adoc[]
+--
+
+[id="network-observability-dns-enhancements-1-7_{context}]
+==== DNS tracking enhancements
+The following enhancements are available for DNS tracking:
+
+* DNS tracking can work with custom port in top of the default DNS port `53`.
```

Comment:
This doesn't sound right.  Suggest:  
If the DNS service maps to a different port than 53, you can specify this DNS tracking port.